chia/data_layer/data_layer_util.py

```diff
@@ -150,7 +150,7 @@ class TerminalNode:
     atom: None = field(init=False, default=None)
 
     @property
-    def pair(self) -> Tuple[bytes32, bytes32]:
+    def pair(self) -> Tuple[Program, Program]:
```

Comment:
I wasn't sure about which way to go to address this. but as far as I can tell, this function isn't used anywhere (at least not triggered by tests, I believe).

Perhaps the mistake was to wrap `self.key` and `self.value` in `Program` and the return type was correct.

Reply:
I think this atom and pair stuff is just garbage at this point.  We'll see what CI says in https://github.com/Chia-Network/chia-blockchain/pull/17063.

Reply:
Looks more intertwined than I thought.  Nodes end up passed to `Program.to()`.  Maybe that shouldn't be the case, but it's at least not a quick delete to resolve this.

Reply:
should I restore the original patch here, to make the return type `Program`?